cocos/editor-support/spine/SkeletonAnimation.h

```diff
@@ -34,7 +34,7 @@
 
 #include <spine/spine.h>
 #include <spine/SkeletonRenderer.h>
-#include "cocos2d.h"
```

Comment:
According spine's license, we can not modify the runtime. It is better if you can send a pull request to https://github.com/EsotericSoftware/spine-runtimes.